cgi-bin/DW/BlobStore/LocalDisk.pm

```diff
@@ -120,7 +120,7 @@ sub delete {
 	my $fqfn = $self->get_location_for_key( $namespace, $key );
 
 	# If the file doesn't exist, pretend success
```

Comment:
I think this comment needs to be updated since it no longer matches the given behavior...